File: RssCollector/feed.cpp
#include "feed.hpp"
#include "downloader.hpp"

namespace models {

	feed::feed(int id, const std::string &url)
	{
			this->id = id;
			this->url = url;
	};

	std::string feed::downloadFeed() const 
	{
		std::string xml = downloader::download(url);
		return xml;
	}
}

File: RssService/models/Feed.py
class Feed():
    def __init__(self, id, url):
        self.id = id
        self.url = url

    def __init__(self, d):
        self.id = d["id"]
        self.url = d["url"]

    def dict(self):
        result = {
            "id": self.id,
            "url": self.url
            }
        return result;
